app/javascript/components/pages/ApartmentNew.js

```diff
@@ -1,9 +1,146 @@
 import React, { Component } from 'react'
+import { Redirect } from 'react-router-dom'
+import { Button } from 'reactstrap'
+import { Form, FormGroup, Input, Label} from 'reactstrap'
 
 export default class ApartmentNew extends Component {
+constructor(props){
+  super(props)
+  this.state ={
+    newListing:{
+      street:"",
+      city:"",
+      state:"",
+      manager:"",
+      email:"",
+      price:"",
+      bedrooms:"",
+      bathrooms:"",
+      pets:"",
+      image:"",
+      user_id:this.props.current_user.id
+    },
+    submitted:false
+  }
+}
```

Comment:
It would be a best practice to put spaces between the `=` and `:` and the following values.